packages/shared/lib/core/network/interfaces/stardust-network-metadata.interface.ts

```diff
@@ -8,7 +8,7 @@ import { IProtocol } from './protocol.interface'
  * necessary for interacting within the context
  * of a particular network.
  */
-export interface INetwork {
+export interface IStardustNetworkMetadata {
```

Comment:
If this is metadata, I don't see how chains fit into it.

Perhaps better to make IPersistedNetwork which can extends IStardustNetworkMetadata with chains